core/color/color_test.go

```diff
@@ -41,3 +41,37 @@ func TestOpposite(t *testing.T) {
 		})
 	}
 }
+
+func TestFromSGFProp(t *testing.T) {
```

Comment:
I'd recommend trying to make this a table test, now that you have the basics of testing down.

Reply:
OK I will do my best!